# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/exim/tail-use.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual tail fixes.

  - Rene Rebe <rene@exactcode.de>

diff -u exim-4.22-orig/scripts/Configure-config.h exim-4.22/scripts/Configure-config.h
--- exim-4.22-orig/scripts/Configure-config.h	2003-08-18 14:52:52.000000000 +0200
+++ exim-4.22/scripts/Configure-config.h	2004-05-03 21:01:25.000000000 +0200
@@ -41,7 +41,7 @@
 
 # Double-check that config.h is complete.
 
-if [ "`tail -1 config.h`" != "/* End of config.h */" ] ; then
+if [ "`tail -n 1 config.h`" != "/* End of config.h */" ] ; then
   echo "*** config.h appears to be incomplete"
   echo "*** unexpected failure in buildconfig program"
   exit 1
